src/index.js

```diff
@@ -108,8 +108,12 @@ async function getContent(octokit, repo, path, ref) {
             repo: repo.name,
             path,
             ref,
+            headers: {
+                // Raw media type necessary for files over 1MB
+                accept: "application/vnd.github.v3.raw",
+            }
         });
-        return Buffer.from(data.content, "base64").toString();
+        return data;
```

Comment:
@calebsevern I'm guessing you tested this on your fork?  I haven't been using this for a while and have just been maintaining.  So if you say it's good and it doesn't break for the original use case I'll take your word on it.

Reply:
@skymoore yeah, I've created a gist to demonstrate the output of `getContent` the exact same before and after https://gist.github.com/calebsevern/2f858938aaa7a6315ff8bdc23b3d64cf

Reply:
Thank you, will get this released then.